### PR TITLE
📝 Document development setup

### PR DESCRIPTION
**Pull request by P3nny**
Based on previous pr by m0ppers 
Add database setup for frontend

Fixes: https://github.com/railslove/recover-backlog/issues/3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,27 +42,43 @@ When a guest checks in, a random ID will be saved on the guest's device. **recov
...
+docker-compose exec postgres psql -U postgres`
+postgres=# insert into frontends (name, url) values('App-Local', 'http://localhost:3333');
+```
+
+Your port might vary depending on your set up (see rcvr-app README).
 
 ### Deployment
 
-- `origin/master` deploys to ___________________ on push
+- `origin/master` deploys to **\*\*\*\***\_\_\_**\*\*\*\*** on push
```
Comment (bonflintstone) [inline README.md:81]: What in the... :flushed: :smile:

File: README.md [hunk 2]
```diff
@@ -89,3 +105,4 @@ And to trigger test events:
 <p align="center">
   Made with 💚 in Cologne
 </p>
+```
```
Comment (salimhb) [inline README.md:108]: Will there be more code here? 😄

**Review by bonflintstone — APPROVED** [review]
Thats what I meant to do

File: README.md (hunk 2 above)
Comment (bonflintstone) [reply README.md:108]: There will always be more code.

**Review by salimhb — APPROVED** [review]

**Review by salimhb — APPROVED** [review]

**Pull request merged**